flink-end-to-end-tests/test-scripts/test_sql_client.sh

```diff
@@ -360,13 +362,14 @@ check_result_hash "SQL Client Kafka" $RESULT "0a1bf8bf716069b7269f575f87a802c0"
 
 echo "Executing SQL: Values -> Elasticsearch (upsert)"
 
-read -r -d '' SQL_STATEMENT_3 << EOF
```

Comment:
Is this cleanup, or ...?

Reply:
yes, I think he already executed the test with set -e enabled. which we should do in the future.

Reply:
If this is indeed just cleanup let's move it into a separate commit as it's entirely unrelated to FLINK-10805.

Reply:
I updated #7023 it should also fix issues covered here.

Reply:
It's not a cleanup. `read` always terminates with a non-zero exit code if it sees `EOF`. It's better to simply use `cat` and this will fix this test to work with the `set -e` command.